src/Security/Security/Az.Security.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.1.1'
+ModuleVersion = '1.1.2'
```

Comment:
Do not update the version - it's calculated during release process

Reply:
Reverted